docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,78 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component we are testing which changes some text depending on `stateprop1` value.
+
+```jsx
+import React, { useReducer } from 'react'
+
+const initialState = {
+    stateprop1: false,
+}
+
+const Reducer = (state = initialState, action) => {
+  switch(action.type) {
+    case "SUCCESS":
+      return {
+        ...state,
+        stateprop1: true,
+      }
+    case "FAILURE":
+      return {
+        ...state,
+        stateprop1: false,
```

Comment:
Not sure if we need a failure case in the example if it is not used in the component, but if so:

```suggestion
        isConfirmed: true,
```